core/modules/proto/ProtoHeaderWrap.h

```diff
@@ -50,8 +50,9 @@ class ProtoHeaderWrap {
     ProtoHeaderWrap() {};
     virtual ~ProtoHeaderWrap() {};
 
-    static std::string wrap(std::string& protoHeaderString);
+    static std::string wrap(std::string const& protoHeaderString);
     static bool unwrap(std::shared_ptr<WorkerResponse>& response, std::vector<char>& buffer);
+    static size_t getProtoHeaderSize();
```

Comment:
Out of curiosity, where is it getting a header to tell you the size? This method, being static, makes it totally unclear.

Reply:
It's a constant defined in the cc file.